src/views/Namespace/List.vue

```diff
@@ -51,33 +51,38 @@
         </div>
       </b-row>
 
-      <b-row
-        v-if="namespacesFiltered && namespacesFiltered.length"
-        align-v="stretch"
-        class="pb-4"
-      >
-        <b-col
-          v-for="n in namespacesFiltered"
-          :key="n.namespaceID"
-          cols="12"
-          md="6"
-          lg="4"
-          xl="3"
-          class="my-2"
+        <b-row
+          v-if="namespacesFiltered && namespacesFiltered.length"
+          align-v="stretch"
+          class="pb-4 mt-3"
         >
-          <namespace-item :namespace="n" />
-        </b-col>
-      </b-row>
-      <b-row
-        v-else
-        class="py-4"
-      >
-        <b-col>
-          <h3 class="text-left">
-            {{ $t('namespace.noResults') }}
-          </h3>
-        </b-col>
-      </b-row>
+          <b-col
+            v-for="n in namespacesFiltered"
+            :key="n.namespaceID"
+            cols="12"
+            md="6"
+            lg="4"
+            xl="3"
+            class="my-2"
+          >
+            <transition-group
+              name="namespace-list"
+              tag="div"
+              class="namespace d-flex flex-wrap flex-column h-100">
```

Comment:
Transition group should not be here, look at webapp-one CAppselector to see where it goes.

Reply:
Each prop in its own row, `>` is also in its own row

Reply:
if I make transition group the parent of b-col, there's a padding I can't get rid of and have to change layout